ci_framework/roles/build_containers/tasks/main.yml

```diff
@@ -35,13 +35,18 @@
 
 - name: Run container_build.sh script
   become: true
+  when: not cifmw_build_containers_run_hotfix | bool
   args:
     chdir: "{{ cifmw_build_containers_basedir }}/artifacts"
   ansible.builtin.shell: |
```

Comment:
Care to use the following syntax:
```YAML
- name: Run container_build.sh script
  become: true
  environment:
    PATH: "{{ cifmw_path }}" # it should have the /usr/local/bin
  ansible.builtin.shell:
    chdir: "{{ cifmw_build_containers_basedir }}/artifacts"
    cmd: >-
      set -o pipefail &&
      bash build_containers.sh 2>&1
      {{ .... }} >
      {{ .... }}
```

Reply:
hmm nope. It's still providing the `chdir` as a separated `args`. Care to update?